### PR TITLE
Additions to Download and corrections to News

### PR DESCRIPTION
**Pull request by souldanger**
- added 'stable' to Ubuntu download instructions
- corrected release News from OpenSUSE to Linux in heading and SEO

### CODE REVIEW

File: download.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ If you are looking for previous versions, please check out the <a href="https://
 </header>
 
 	wget -qO - https://downloads.iridiumbrowser.de/ubuntu/iridium-release-sign-01.pub|sudo apt-key add -
-    sudo apt-add-repository 'https://downloads.iridiumbrowser.de/ubuntu main'
+    sudo apt-add-repository 'https://downloads.iridiumbrowser.de/ubuntu stable main'
```
Comment (longsleep) [inline download.md]: This change is not correct.

Best possible command:
```
sudo apt-add-repository 'deb [arch=amd64] https://downloads.iridiumbrowser.de/deb/ stable main'
```

This also retains compatibility with
```
cat <<EOF | sudo tee /etc/apt/sources.list.d/iridium-browser.list
deb [arch=amd64] https://downloads.iridiumbrowser.de/deb/ stable main
#deb-src https://downloads.iridiumbrowser.de/deb/ stable main
EOF
sudo apt-get update
sudo apt-get install iridium-browser
```
Which is the recommended way to enable iridium repository and install on all debian based distros.

**Comment by souldanger** [comment]
Hi Simon
thanks a bunch for letting me know! I am obviously not a Penguin... :-)
I have updated accordingly, please see: [https://test.iridiumbrowser.de/download.html](https://test.iridiumbrowser.de/download.html)

File: download.md [hunk 2]
```diff
@@ -62,9 +62,11 @@ If you are looking for previous versions, please check out the <a href="https://
 	<p>current version 54.0</p>
 </header>
 
-	wget -qO - https://downloads.iridiumbrowser.de/ubuntu/iridium-release-sign-01.pub|sudo apt-key add -
```
Comment (longsleep) [inline download.md]: Please keep instructions to grab and add the key. Without that the rest will not work.

**Comment by souldanger** [comment]
THANKS!
updated accordingly. created new pull request see [https://github.com/iridium-browser/iridium-browser/pull/119](https://github.com/iridium-browser/iridium-browser/pull/119)
PLEASE merge if you can, as Ryan is still overseas.
Cheers!

**Pull request closed without merging**